File: python/linear_search.py
array_to_search_through = []
for number in range(1, 1001):
    array_to_search_through.append(number)

def linear_search(value_to_find, array_to_search_through):
    # your code here
    for value in range( len(array_to_search_through)):
        if value_to_find == array_to_search_through[value]:
            return value

def linear_search_global(value_to_find, array_to_search_through):
    # your code here
    myValue = []
    for value in range( len(array_to_search_through)):
        if value_to_find == array_to_search_through[value]:
            myValue.append(value)
    return myValue

   

File: js/linearSearch.js
const arrayToSearchThrough = [];
for (let i = 1; i <= 1000; i++) {
    arrayToSearchThrough.push(i);
}

exports.linearSearch = function(valueToFind, arrayToSearchThrough) {
    // your code here
    let indexOfArr ;
    for(let i = 0; i < arrayToSearchThrough.length; i++){
        if(arrayToSearchThrough[i] === valueToFind){
            indexOfArr = i;
        }
    }
    console.log(indexOfArr)
    return indexOfArr;
};

exports.linearSearchGlobal = function(valueToFind, arrayToSearchThrough) {
    // your code here
    let indexOfArr = []
    for(let i = 0; i < arrayToSearchThrough.length; i++){
        if(arrayToSearchThrough[i] === valueToFind){
            indexOfArr.push(i);
        }
    }
    console.log(indexOfArr)
    return indexOfArr
};